### DISCUSSION TITLE
How do I get type annotation by highlighting+keybinding?

### DISCUSSION THREAD

**Discussion by shoaibkhanz** [LazyVim/LazyVim#787] [Q&A] [answered]
Hi, I have been using lazynvim for sometime but still new and mostly use python. The `cmp lsp` does provide auto completion and access to docstrings and types. But I want to know, how can I access type annotation by highlighting over it just like in vscode ?

I found that coc.nvim  does have this feature, but not sure how to install it in lazy config or perhaps cmp does it and I am just unaware how to activate it.

![68747470733a2f2f616c66732e6368696775612e636e2f6469616e796f752f646174612f706c6174666f726d2f64656661756c742f32303232303830312f323032322d30382d303125323030322d31342d30332e323032322d30382d303125323030325f31355f31362e676966](https://github.com/LazyVim/LazyVim/assets/6944091/2c46fd21-edc9-49f6-9c69-479560a01715)

Any help would be highly appreciated :)

**Comment by folke** [comment, marked as answer]
You go over the thing you want to see the definition and press `K`
